themes/bootstrap/static/css/multiselect.css

```diff
@@ -0,0 +1,151 @@
+/** Multiselect component adapted from
+    https://webdevtrick.com/html-combobox-with-javascript/
+  */
+
```

Comment:
The combo box seems to have the same height as the search box. Would it be possible to make them a bit narrower? Could we display a placeholder text like `select language` (or just `select...`, if there is not enough space)?